PRFAQ-for-launch.md

```diff
@@ -0,0 +1,47 @@
+**Wardley Mapping Foundation launched to promote strategic planning based on situational awareness, appropriate doctrines, and the evolution of value chains**
+
+_London, September 1st, 2022._ A community of practitioners of the Wardley Mapping strategy techniques created by researcher Simon Wardley announce that they have formed the Wardley Mapping Foundation (WMF) to create a global and open mechanism to coordinate and promote the development of Wardley Mapping.
+
+Business and government decision makers need to have situational awareness and understanding of doctrines, the climate they are operating in, and the evolution and diffusion of ideas and technologies.
+
+Wardley Mapping is based on decades of research into business successes and failures and has been used to guide successful strategic decisions by startups, organizations of all sizes, and governments around the world. Maps describe position and movement, and Wardley Maps show how value chains evolve, with guidance on effective use of doctrine to make strategic decisions, compete in and disrupt markets.
+
+Maps in general are simplified models of a landscape, created for a purpose. Wardley Maps are relatively easy to read and follow, but it takes time and practice to build the body of knowledge required to lead an interactive mapping exercise and create a series of maps and associated strategic moves. The WMF was created by a group of practitioners with support from Simon Wardley, in order to grow awareness and acceptance of the methodology, develop tools and training materials, and to endorse events and related products via the WMF brand. The goal is for Wardley Mapping to become a community driven endeavor, that is supported by, but no longer dependent upon, Simon Wardley himself.
```

Comment:
see comment in https://github.com/Wardley-Mapping-Foundation/wmf-operating-model/pull/24#issuecomment-1136042093